x-pack/plugins/security_solution/public/management/components/endpoint_execute_action/execute_action_host_response_output.tsx

```diff
@@ -84,45 +81,30 @@ const ExecutionActionOutputAccordion = memo<ExecuteActionOutputProps>(
 ExecutionActionOutputAccordion.displayName = 'ExecutionActionOutputAccordion';
 
 export interface ExecuteActionHostResponseOutputProps {
-  action: MaybeImmutable<ActionDetails>;
-  agentId?: string;
+  outputContent: ResponseActionExecuteOutputContent;
   'data-test-subj'?: string;
   textSize?: 's' | 'xs';
 }
 
 export const ExecuteActionHostResponseOutput = memo<ExecuteActionHostResponseOutputProps>(
-  ({ action, agentId = action.agents[0], 'data-test-subj': dataTestSubj, textSize = 'xs' }) => {
-    const outputContent = useMemo(
-      () =>
-        action.outputs &&
-        action.outputs[agentId] &&
-        (action.outputs[agentId].content as ResponseActionExecuteOutputContent),
-      [action.outputs, agentId]
-    );
-
-    if (!outputContent) {
-      return <></>;
-    }
-
-    return (
-      <EuiFlexItem data-test-subj={dataTestSubj}>
-        <EuiSpacer size="m" />
-        <ExecutionActionOutputAccordion
-          content={outputContent.stdout}
-          isTruncated={outputContent.stdout_truncated}
-          initialIsOpen
-          textSize={textSize}
-          type="output"
-        />
-        <EuiSpacer size="m" />
-        <ExecutionActionOutputAccordion
-          content={outputContent.stderr}
-          isTruncated={outputContent.stderr_truncated}
-          textSize={textSize}
-          type="error"
-        />
-      </EuiFlexItem>
-    );
-  }
+  ({ outputContent, 'data-test-subj': dataTestSubj, textSize = 'xs' }) => (
+    <EuiFlexItem data-test-subj={dataTestSubj}>
+      <EuiSpacer size="m" />
+      <ExecutionActionOutputAccordion
+        content={outputContent.stdout.trim().length ? outputContent.stdout.trim() : undefined}
+        isTruncated={outputContent.stdout_truncated}
+        initialIsOpen
+        textSize={textSize}
+        type="output"
+      />
+      <EuiSpacer size="m" />
+      <ExecutionActionOutputAccordion
+        content={outputContent.stderr.trim().length ? outputContent.stderr.trim() : undefined}
```

Comment:
Not sure you should be `.trim()`'ing the `stdout` and `stderr`. I think we should displayed it "as is" received.

Reply:
Yeah, I wasn't sure about this. I'll show it as it is without the `trim()`

Reply:
done 084eaa0a2fdfa12fe1326a2937a3f4cc3b762c08